app/controllers/schools/manual_readings_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Schools
+  class ManualReadingsController < ApplicationController
+    load_and_authorize_resource :school
+    load_and_authorize_resource through: :school
+
+    before_action :set_breadcrumbs
+
+    include SchoolProgress
+
+    def show
+      @current_target = @school.current_target
+      @months = manual_months
+      existing_months = @school.manual_readings.pluck(:month)
+      @months.values.flatten.uniq.each do |month|
+        @school.manual_readings.build(month:) unless existing_months.include?(month)
+      end
+
+      @readings = @school.manual_readings.sort_by(&:month)
+    end
+
+    def update
+      @school.update(resource_params)
+      if @school.save
+        redirect_to school_manual_readings_path(@school), notice: t('common.saved')
+      else
+        render :show
+      end
+    end
+
+    private
+
+    def manual_months
```

Comment:
Could we remove the reliance on having an existing target? I want to use this manually supplied data to improve long term use pages for schools with limited data. Also, we might want to prompt schools to add this before they've set a target. Maybe just default to allowing up to a year of data?

Reply:
Ideally we want at least a year (13 months to allow for comparisons) for each fuel type. This could be used for targets or elsewhere.

For targets, while we have schools that currently have limited data and data, when setting new targets you shouldn't be able to set one in the past, so the start date for those shouldn't be earlier than the current month.

Reply:
> Ideally we want at least a year (13 months to allow for comparisons) for each fuel type. This could be used for targets or elsewhere.
> 

wonder if allowing input for 2 years of data would be better?